apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -1351,7 +1351,7 @@ export async function getPaperHandlerStateMachine({
   driver: PaperHandlerDriverInterface;
   patConnectionStatusReader: PatConnectionStatusReaderInterface;
   clock?: Clock;
-}): Promise<Optional<PaperHandlerStateMachine>> {
+}): Promise<PaperHandlerStateMachine> {
```

Comment:
Unrelated to flaky tests but I noticed this `Optional` was never returning `undefined`.